src/services/user/updateUser.js

```diff
@@ -36,6 +36,18 @@ const updateUser = ayonApi.injectEndpoints({
       invalidatesTags: () => ['user'],
       transformErrorResponse: (res) => res.data,
     }),
+    updateUserAvatar: build.mutation({
+      query: ({ name, avatarUrl }) => ({
+        url: `/api/users/${name}/avatar`,
+        method: 'PUT',
+        body: { avatarUrl },
+      }),
+      invalidatesTags: (result, error, { name }) => [
+        { type: 'user', id: name },
+        { type: 'user', id: 'LIST' },
```

Comment:
I know it's a WIP but FYI 'LIST' will "theoretically" invalidate every single user, when maybe we only want to invalidate the specific user.

I say theoretically, but that all depends if the provided tags have been set up correctly.


https://redux-toolkit.js.org/rtk-query/usage/automated-refetching#advanced-invalidation-with-abstract-tag-ids

Reply:
Thanks for info, after some discussion it was recommended to use axios for update avatar, thats what currently implemented.